### PR TITLE
fix: [M3-8763] - fix flaky `DatabaseBackups.test.tsx` in coverage job

### PR DESCRIPTION
**Pull request by abailly-akamai**
## Description 📝
Attempting to fix an annoyance with this test **_only_** failing on the `coverage:summary` job in the CI, resulting a failing step. While this is non blocking, it prevents the coverage job from running and therefore update our score.

It is hard to pinpoint when this started happening, but every PR is experiencing this issue so it needs to be fixed.
I am forcing the  `current_branch` to run, so we can still see if `yarn coverage:summary` actually passes without issues on the current branch in case `base_branch` is flaky

## Changes  🔄
- Improve `DatabaseBackups.test.tsx` test
- force `current_branch` coverage step to run

## Locally
- run `yarn test DatabaseBackups.test.tsx` and verify it passes
- run `yarn coverage:summary` confirmed no flakes/passes 100%

## CI
- confirm coverage > base_branch > `Run Current Branch Coverage` succeeds
- confirm coverage > current_branch > `Run Current Branch Coverage` succeeds

## As an Author I have considered 🤔

*Check all that apply*

- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [x] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [x] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: .github/workflows/coverage.yml [hunk 1]
```diff
@@ -47,7 +47,9 @@ jobs:
           path: ref_code_coverage.txt
 
   current_branch:
-    if: github.event.pull_request.draft == false
+    # We want to make sure we only run on draft, but also should run even if the base branch coverage job fails
```
Comment (mjac0bs) [inline .github/workflows/coverage.yml:50]: > we only run on draft

But draft == false? Is this supposed to be "we only run on open, not draft, PRs", or am I misunderstanding something?

**Review by mjac0bs — APPROVED** [review]
Did not observe flake locally or in CI and looked good other than my one docs comment.

File: .github/workflows/coverage.yml (hunk 1 above)
Comment (abailly-akamai) [reply .github/workflows/coverage.yml:50]: oh comment is wrong i meant the opposite 🤦 thx for catching

Updating and rerunning, will confirm no flake, too

**Comment by abailly-akamai** [comment]
cool, current_branch job is passing consistently, good to merge

**Pull request merged**